terrain_navigation_ros/scripts/global_postion_relay.py

```diff
@@ -0,0 +1,80 @@
+"""
```

Comment:
If this is a node, should it have a shebang and be marked executable?

Reply:
Yep - oversight. Will add.

Reply:
Updated in https://github.com/ethz-asl/terrain-navigation/pull/27/commits/619c4bbfca93645ded3ef5d22dd156eef1823f4a